src/t8_cmesh/t8_cmesh_examples.cxx

```diff
@@ -1335,7 +1335,7 @@ t8_cmesh_new_hypercube_pad_ext (const t8_eclass_t eclass, sc_MPI_Comm comm, cons
     for (t8_gloidx_t tree_x = 0; tree_x < polygons_x; tree_x++) {
       t8_cmesh_set_tree_vertices (cmesh, tree_x + offset, vertices, 2);
       /* Update vertices for next tree */
-      t8_axy (vertices, vertices + 3, 1.0);
+      t8_axy (vertices + 3, vertices, 1.0);
       t8_axpy (line_dir, vertices + 3, 1.0);
```

Comment:
I do not think that the order was wrong. Please double check.
As i understand you want to copy the values from `vertices` over to `vertices + 3`.
So `vertices + 3` is the output.
As such it should be the 2nd argument.

```C++
/** Compute Y = alpha * X
 * \param [in]  vec_x  A 3D vector.
 * \param [out] vec_z  On output set to \a alpha * \a vec_x.
 * \param [in]  alpha  A factor.
 */
void
t8_axy (const double vec_x[3], double vec_y[3], const double alpha);
```

Reply:
No, the order was wrong. I want to copy the values from vertices + 3 to vertices. 
In the next line you see how the coords in vertices + 3 are shifted in line_dir.